### PR TITLE
:sparkles: feature: add XML to context.

### PR DESCRIPTION
**Pull request by efectn**
**Please provide enough information so that others can review your pull request:**

**Explain the *details* for making this change. What existing problem does the pull request solve?**

**Commit formatting** 

Use emojis on commit messages so it provides an easy way of identifying the purpose or intention of a commit. Check out the emoji cheatsheet here: https://gitmoji.carloscuesta.me/

### CODE REVIEW

File: ctx.go [hunk 1]
```diff
@@ -736,6 +731,18 @@ func (c *Ctx) JSONP(data interface{}, callback ...string) error {
 	return c.SendString(result)
 }
 
+// XML converts any interface or string to XML.
+// This method also sets the content header to application/xml.
+func (c *Ctx) XML(data interface{}) error {
```
Comment (Trim21) [inline ctx.go:736]: use `any` considering we are already using go1.18?

Comment (efectn) [reply ctx.go:736]: This is v2

Comment (Trim21) [reply ctx.go:736]: oh sorry 😥

Comment (efectn) [reply ctx.go:736]: No problem. I'll convert it to any when merging master into the v3-beta.

File: app.go [hunk 2]
```diff
@@ -323,6 +325,13 @@ type Config struct {
 	// Default: json.Unmarshal
 	JSONDecoder utils.JSONUnmarshal `json:"-"`
 
+	// XMLEncoder set by an external client of Fiber it will use the provided implementation of a
+	// XMLMarshal
+	//
+	// Allowing for flexibility in using another XML library for encoding
+	// Default: xml.Marshal
+	XMLEncoder utils.JSONMarshal `json:"-"`
```
Comment (ReneWerner87) [inline app.go:333]: you mean utils.XMLMarshal

**Review by ReneWerner87 — CHANGES_REQUESTED** [review]

File: app.go (hunk 2 above)
Comment (efectn) [reply app.go]: Done

**Review by ReneWerner87 — APPROVED** [review]

**Pull request merged**